src/templates/markdownTemplate.tsx

```diff
@@ -14,6 +14,24 @@ type markdownBaseProps = {
   data: { mdx: markdownRemarkResult }
   pageContext: markdownPageContext
 }
+const getChildrenAsString = (
+  props: React.HtmlHTMLAttributes<HTMLHeadingElement>
+) => {
+  if (typeof props.children === 'string') {
+    return props.children.replace(/(\s|\.)+/g, '-').toLowerCase()
+  }
+  // TODO if we'd like headers with children to still have IDs
+  // then we can test and uncomment this code
+  // if (props.children) {
```

Comment:
what would be an example of headers with children? i think we can probably remove this

Reply:
If we had wanted code in a header for example